pages/docs/event-analytics/conversion-rates.mdx

```diff
@@ -0,0 +1,19 @@
+# How are conversion rates computed?
+
+Conversion rates in Hashboard are measured using the following formula:
+
+$$$
+rate = \frac{converted\_count}{started\_count}
```

Comment:
please rename to `entered_funnel` and `converted_funnel` to match the columns names of the funnel conversion table.